app/components/place-holder.js

```diff
@@ -118,7 +119,8 @@ export default Ember.Component.extend({
             r.organizations = source.lists.organizations;
             return r;
         }));
-        this.set('widgetType', 'donut-chart');
+        this.set('widgetType', 'generic-chart');
```

Comment:
Maybe we make a separate component for each charting library this would be something like `this.set('widgetType', 'c3-chart');` but still changeable in the settings panel to `dimple` or `NVD3`. There will be some components that simply aren't charts and shouldn't be in the same file ase a component that is for rendering charts.